processor/src/test/java/org/mapstruct/ap/test/builder/simple/SimpleImmutablePerson.java

```diff
@@ -80,5 +97,19 @@ public Builder city(String city) {
             this.city = city;
             return this;
         }
+
+        public Builder address(String address) {
+            this.address = address;
+            return this;
+        }
+
+        public List<String> getChildren() {
+            throw new UnsupportedOperationException( "This is just a marker method" );
+        }
+
+        public Builder addChild(String child) {
```

Comment:
have a look at the class `Nouns`.. that should do the trick of preferring `addChild` over `address`..

Reply:
I know about it, but that is from the internal packages.